packages/db/prisma/seed/DataGenerator.ts

```diff
@@ -184,7 +184,8 @@ export class DataGenerator {
       let fromId: string;
 
       if (rollup) {
-        fromId = ROLLUP_ADDRESSES[rollup];
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        fromId = ROLLUP_ADDRESSES[rollup]!;
```

Comment:
```suggestion
        fromId = ROLLUP_ADDRESSES[rollup];
```
If you apply the changes to ROLLUP_ADDRESSES, this assertion won't be necessary

Reply:
TS complains as `ROLLUP_ADDRESSES[rollup] may still be `undefined`

I refactor this in order to remove the `no-null-assertion` comment